fec.config.js

```diff
@@ -0,0 +1,29 @@
+const { resolve } = require('path');
+
+module.exports = {
+    appUrl: '/insights/patch',
+    debug: true,
+    useProxy: process.env.PROXY === 'true',
+    proxyVerbose: true,
+    interceptChromeConfig: false,
+    plugins: [],
+    ...(process.env.HOT ? { hotReload: process.env.HOT === 'true' } : { hotReload: true }),
+    ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+    moduleFederation: {
+        exclude: ['react-router-dom'],
```

Comment:
I'm very suspicious of this. Do we really need to add this to every app, even if the react router version is v6?

I think we only need to do this if the version is not v6, like policies for example.

Reply:
I mean both `expose` and `shared`.

Reply:
Not all apps have upgraded to v6. Can this be a reason to have this in place still?

Reply:
I understand that once all are migrated, we were expected to remove this.

Reply:
@mkholjuraev I really think we only need to do this in apps that are not migrated, and not in all till all are migrated. Chrome is sharing its version, which is v6. This would cause issues in apps with v5 and there we can add these definitions to make the app share its version instead of chrome.

Reply:
If without this the app still works proper, we probably don't need it.

Reply:
I have tried it out. I have removed both exclude&shared configs related to router, and the UI does not work with this error: **useRoutes() may be used only in the context of a <Router> component.**

Reply:
Call Stack:
 invariant
  vendors-node_modules_patternfly_react-component-groups_node_modules_clsx_dist_clsx_mjs-node_m-af6625.js:12071:11
 useRoutesImpl
  vendors-node_modules_patternfly_react-component-groups_node_modules_clsx_dist_clsx_mjs-node_m-af6625.js:34809:102
 useRoutes
  vendors-node_modules_patternfly_react-component-groups_node_modules_clsx_dist_clsx_mjs-node_m-af6625.js:34804:10
 Routes
  vendors-node_modules_patternfly_react-component-groups_node_modules_clsx_dist_clsx_mjs-node_m-af6625.js:35693:10
 renderWithHooks
  chrome-root.b25b7d405e94b287.js:37887:18
 mountIndeterminateComponent
  chrome-root.b25b7d405e94b287.js:41651:13
 beginWork
  chrome-root.b25b7d405e94b287.js:43164:16
 HTMLUnknownElement.callCallback
  chrome-root.b25b7d405e94b287.js:25746:14
 Object.invokeGuardedCallbackDev
  chrome-root.b25b7d405e94b287.js:25795:16
 invokeGuardedCallback
  chrome-root.b25b7d405e94b287.js:25859:31